python/pyproject.toml

```diff
@@ -17,7 +17,7 @@ dependencies = ["requests", "tqdm", "numpy", "IPython", "setproctitle"]
 
 [project.optional-dependencies]
 runtime_common = ["aiohttp", "decord", "fastapi",
-    "hf_transfer", "huggingface_hub", "interegular", "modelscope",
+    "hf_transfer", "huggingface_hub", "interegular", "nvidia-modelopt", "modelscope",
```

Comment:
Please place `nvidia-modelopt` in `srt`, not in `common`, as other hardware like AMD doesn't require it.

Reply:
As wished.